src/Tasks.UnitTests/Microsoft.Build.Tasks.UnitTests.csproj

```diff
@@ -13,7 +13,6 @@
 
   <ItemGroup>
     <PackageReference Include="System.Security.Principal.Windows" />
-    <PackageReference Include="System.Xml.XPath" />
```

Comment:
Can you remove the packagereference update in eng\packages.props for System.Xml.XPath?